src/ui/Facet/FacetSearch.ts

```diff
@@ -71,8 +71,8 @@ export class FacetSearch implements IFacetSearch {
   /**
    * Position the search results at the footer of the facet.
    */
-  public positionSearchResults(nextTo: HTMLElement = this.search) {
-    this.facetSearchElement.positionSearchResults(this.root, this.facet.element.clientWidth, nextTo);
+  public positionSearchResults() {
```

Comment:
Since the `FacetSearch` class is publicly exposed via the `Facet` class, removing the argument would be a breaking change. How important is the `nextTo` argument? And what are the impacts of removing it?

Reply:
Hmm, good point.

The `nextTo` argument defined under which element the search results are going to appear. However, it wasn't necessary here anymore since I removed the `PopupUtils` class that was used to position the element under `nextTo`, effectively making the popup always appear under the search box.